### PR TITLE
Apple Pay Continued - Online Payment Cryptogram

### PR DESCRIPTION
**Pull request by scottbartell**
include the apple pay cryptogram in the tokenize response

### CODE REVIEW

File: spec/api/request_body_spec.rb [hunk 1]
```diff
@@ -148,6 +148,41 @@
...
+          amount: 4224,
+          customer_id: "extid123",
+          payment_account_id: "paymentacct123",
+          order_id: "SomeOrder123",
+          expiry_month: "8",
+          expiry_year: "2018"
+        )
+      end
+
+      it "sets the cardholder authentication value" do
```
Comment (ChrisVoxland) [inline spec/api/request_body_spec.rb]: `does not set the cardholder authentication value` or `sets the cardholder authentication value to nil`

**Comment by ChrisVoxland** [comment]
👍 on green once test description is updated

**Comment by scottbartell** [comment]
@ChrisVoxland spec description updated - good find!

**Comment by scottbartell** [comment]
On a kind of related note, I noticed that we have no specs actually testing the generated XML. This might not be too big of a deal because the test requests that we make to the Vantiv test API will likely start to fail if we make breaking changes to the representers/generated XML. However, some cases might fall through the cracks.

For instance, because we don't have an end-to-end test for Apple Pay (as we've learned, it's not actually possible for us to have one right now), I didn't have a failing test when the authentication_value/onlinePaymentCryptogram was missing from the request -- this is why I added some specs testing the actual XML request being made.

**Pull request merged**